### PR TITLE
added chapter2 content

### PR DESCRIPTION
**Pull request by stschiff**
This is the first entry of Chapter 2 as it was sent for internal circulation. Note that I had created the branch from Version v0.1.1, so some details changed in v0.1.2 are missing here, but merging will take care of this, of course.

### CODE REVIEW

File: chapter_2.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+
+II. 9. Each Member State should consider maintaining untouched, partially or totally, a certain number of archaeological sites of different periods in order that their excavation may benefit from improved techniques and more advanced archaeological knowledge. On each of the larger sites now being excavated, in so far as the nature of the land permits, well defined `witness' areas might be left unexcavated in several places in order to allow for eventual verification of the stratigraphy and archaeological composition of the site.
```
Comment (nevrome) [inline chapter_2.md:7]: `witness' -> “witness”

**Review by nevrome — APPROVED** [review]
Great!

File: chapter_2.md (hunk 1 above)
Comment (stschiff) [reply chapter_2.md:7]: Ah yes OK, Thanks.

**Comment by stschiff** [comment]
@scarlhoff any additional comments?

**Pull request merged**